public/app/features/variables/pickers/shared/VariableOptions.tsx

```diff
@@ -63,12 +63,17 @@ export class VariableOptions extends PureComponent<Props> {
 
     return (
       <li key={`${option.value}`}>
-        <a role="checkbox" aria-checked={option.selected} className={highlightClass} onClick={this.onToggle(option)}>
+        <button
+          role="checkbox"
+          aria-checked={option.selected}
+          className={`${highlightClass} ${noStyledButton}`}
```

Comment:
think we should use `classnames` here instead of interpolated strings

Reply:
Hi @ashharrison90. In this case, `highlightClass`was already there and its value is set according to a condition so I think this shouldn't be changed. Regarding `noStyledButton`, we could either let it be or change it to use `classnames`. What do you think is better?

Reply:
yeah i think that's fine, i'm saying use `classnames` to apply multiple classes instead of using string interpolation

e.g. `classNames={classnames(highlightClass, noStyledButton)}`

Reply:
Ah, ok. I didn't know about that 🤔  Thank you @ashharrison90!!!

Reply:
Aren't we using `cx` from emotion for this kind of stuff?

Reply:
@Clarity-89 last thing i remember is we decided we can use either

`cx` performance is much worse than `classnames`: https://github.com/grafana/grafana/issues/38698

Reply:
Ah, makes sense. Didn't know `cx` had such a poor performance.

Reply:
the only thing we have to be careful of is when composing multiple emotion `css` classes... tbh if the `cx` performance wasn't like 4-5 times worse we should definitely be using `cx` everywhere 😅